packages/ckeditor5-block-quote/docs/features/block-quote.md

```diff
@@ -2,17 +2,28 @@
 category: features
 menu-title: Block quote
 ---
+{@snippet features/block-quote-source}
 
 # Block quote
 
-The {@link module:block-quote/blockquote~BlockQuote} feature allows you to easily include block quotations or pull quotes in the rich-text content. This provides an attractive way to draw the readers' attention to selected parts of text. It also helps organize the content in a structured, elegant way and to manage the flow better.
+The {@link module:block-quote/blockquote~BlockQuote} feature allows you to easily include block quotations or pull quotes in the rich-text content. A block quote may be used to refer to what other authors said, support your idea or present different points of view that may not necessarily be in line with the document you write, but should be included. This provides an attractive way to draw the readers' attention to selected parts of text and enriches the reading experience with additional data. It also helps organize the content in a structured, elegant way and to manage the flow better.
 
 ## Demo
 
 Use the editor below to see the block quote plugin in action. You can also precede the quotation with the `>` inline code (followed by a space) to format it on the go thanks to the {@link features/autoformat autoformatting} feature.
 
 {@snippet features/block-quote}
 
+## Nesting block quotes
+
+Sometimes it is necessary to nest a block quote into another block quote. This may be the case for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it. To use this functionality, simply click where you need the quote nested and insert it just like a regular one.
```

Comment:
```suggestion
Sometimes it is necessary to nest a block quote into another block quote. This may be the case for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it. To use this functionality, simply click where you need the quote nested and insert it just like a regular one.
```

Actually I would start it differently - from something like "Starting from version N.N.N, CKEditor 5 supports nesting a block quote in another one." We need to have the version though - this will be known this Wed so you'll need to remember about it.

Reply:
Addressed in commit.